ajax/group_values.php

```diff
@@ -12,21 +12,12 @@
    $groups_id_filtred = $PluginEscaladeGroup_Group->getGroups($ticket_id);
 
 if (count($groups_id_filtred) > 0) {
-   $myarray = [];
-   foreach ($groups_id_filtred as $groups_id => $groups_name) {
-      $myarray[] = $groups_id;
-   }
-   $newarray = implode(", ", $myarray);
-   $condition = " id IN ($newarray)";
-
+   $condition = ['id' => array_keys($groups_id_filtred)];
 } else {
-   $condition = "1=0";
+   $condition = [new QueryExpression('1 = 0')];
```

Comment:
don't know if this work
```suggestion
   $condition = [false];
```

Reply:
```php
$it = new DBmysqlIterator(null);
$it->buildQuery(
   [
      'FROM' => 'mytable',
      'WHERE' => [false],
   ]
);
echo($it->getSql());
//SELECT * FROM `mytable` WHERE ()
```

Reply:
It works with `['false']` but not with `[false]`